### PR TITLE
Fix post data and rewrite internal URLs

### LINKED ISSUE #9: Rewrite links
We might need to rewrite some (absolute) links as we're using a headless WP instance and/or if we import existing content from blog.mozilla.org.

### LINKED ISSUE #34: Rename `postUrl` to `permalink`

### PR DESCRIPTION
**Pull request by willdurand**
Fixes #9 
Fixes #34

### CODE REVIEW

File: src/wordpress.js [hunk 1]
```diff
@@ -6,11 +6,46 @@ const fetch = require('node-fetch');
...
+const DEFAULT_WORDPRESS_BASE_URL = 'https://mozamo.wpengine.com';
+
 async function getNumPages(endPoint) {
   const result = await fetch(endPoint, { method: 'HEAD' });
   return result.headers.get('x-wp-totalpages') || 1;
 }
 
+const fixInternalURLs = (content) => {
+  return content.replace(
+    /https?:\/\/mozamo\.wpengine\.com/g,
```
Comment (muffinresearch) [inline src/wordpress.js:19]: Did you see if changing the site URL changes the output? I guess it'll be worth having this anyway, but that might save on the replacements? It's on general settings.


General_Settings_‹_Mozilla_Add-ons_Blog_—_WordPress

**Review by muffinresearch — APPROVED** [review]
LGTM r+wc

File: src/wordpress.js (hunk 1 above)
Comment (willdurand) [reply src/wordpress.js:19]: @muffinresearch please see https://github.com/mozilla/addons-blog/issues/9#issuecomment-808518332, I tried but this setting is used in the editor too...

Comment (muffinresearch) [reply src/wordpress.js:19]: Sorry I missed that.

**Pull request merged**